web/src/main/java/org/cbioportal/web/SampleController.java

```diff
@@ -244,7 +256,7 @@ private boolean usingAuth() {
             List<Sample> samples;
 
             if (interceptedSampleFilter.getSampleListIds() != null) {
-                samples = sampleService.fetchSamples(interceptedSampleFilter.getSampleListIds(), projection.name());
+                samples = instance.fetchSamplesInner(interceptedSampleFilter.getSampleListIds(), projection.name());
```

Comment:
This is fine, but why not cache this for meta projections as well? (Line 244)

Reply:
Ok, never mind, meta is super fast.

Reply:
meta projections is fast, so improvement by caching this could be minimal, as of now.